cmd/rootCmd.go

```diff
@@ -36,7 +36,6 @@ var rootCmd = &cobra.Command{
 
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", err)
```

Comment:
Do we not longer print out the error?

Reply:
[Command.SilenceErrors](https://godoc.org/github.com/spf13/cobra#Command) toggles this, so cobra will already print any error.